### ISSUE TITLE
NPE onEnable

### ISSUE THREAD

**Issue by SlimeDog** [mcmonkeyprojects/Sentinel#406]
The output of command `/version` on my server is: 
```
> version
[10:53:47 INFO]: This server is running Paper version 1.21.3-10-master@92131ad (2024-11-01T21:30:00Z) (Implementing API version 1.21.3-R0.1-SNAPSHOT)
You are running the latest version
Previous version: 1.21.3-9-d19be64 (MC: 1.21.3)
```
The output of command `/version citizens` on my server is: 
```
> version Citizens
[10:54:43 INFO]: Citizens version 2.0.36-SNAPSHOT (build 3626)
[10:54:43 INFO]: Website: https://www.citizensnpcs.co
[10:54:43 INFO]: Author: fullwall
```
The output of command `/version sentinel` on my server is: 
```
> version Sentinel
[10:54:47 INFO]: Sentinel version 2.9.1-SNAPSHOT (build 525)
[10:54:47 INFO]: Author: mcmonkey
```
`Sentinel/config.yml` file is: 
```
grep -v -e ' *#' plugins/Sentinel/config.yml

sentinel defaults:
    close chase target: true
    ranged chase target: false
    damage: -1
    armor: -1
    health: 20
    invincible: false
    attack rate: 30
    heal rate: 100
    protected: true
    protect from range: 0
    fightback: true
    needs ammo: false
    safe shot: true
    respawn time: 100
    enemy drops: false
    enemy target time: 6000
    speed: 1.5
    autoswitch: false
    accuracy: 0
    realistic: false
    reach: 4.5
    projectile range: 100
    avoid range: 10
    runaway: false
    greet range: 10
    greet rate: 100
    reaction slowdown: 0
    allow knockback: true
    death xp: 0

random:
    run away time: 1200
    spectral sound: ENTITY_VILLAGER_YES
    workaround damage: false
    do native attacks: true
    workaround drops: false
    workaround bukkit events: false
    workaround entity chase pathfinder: true
    death messages: false
    shoot speed minimum: 20
    enforce damage: false
    skull allowed: true
    clever ticks: 10
    ignore invisible targets: true
    no guard damage: true
    enhance los traces: true
    guard follow distance:
        selection range: 4
        minimum: 7
    arrow cleanup time: 200
    block sunburn: true
    retain target: false
    prevent explosion block damage: true
    auto correct pathfinder mode: true
    allow armor stand targets: false

update rate: 10
```

On server start-up, onEnable Sentinel:
```
[09:36:58] [Server thread/INFO]: [Sentinel] Enabling Sentinel v2.9.1-SNAPSHOT (build 525)
[09:36:58] [Server thread/INFO]: [Sentinel] Sentinel loading... 
[09:36:58] [Server thread/INFO]: [Sentinel] Running on java version: 23.0.1
[09:36:59] [Server thread/INFO]: [Sentinel] Vault linked! Group targets will work.
[09:36:59] [Server thread/INFO]: [Sentinel] Sentinel found WorldGuard! Adding support for it!
[09:36:59] [Server thread/WARN]: java.lang.NullPointerException 
[09:36:59] [Server thread/WARN]:        at java.base/java.util.Objects.requireNonNull(Objects.java:220)
[09:36:59] [Server thread/WARN]:        at java.base/java.lang.invoke.DirectMethodHandle.checkBase(DirectMethodHandle.java:546)
[09:36:59] [Server thread/WARN]:        at java.base/jdk.internal.reflect.MethodHandleIntegerFieldAccessorImpl.getInt(MethodHandleIntegerFieldAccessorImpl.java:79)
[09:36:59] [Server thread/WARN]:        at java.base/jdk.internal.reflect.MethodHandleIntegerFieldAccessorImpl.get(MethodHandleIntegerFieldAccessorImpl.java:55) 
[09:36:59] [Server thread/WARN]:        at java.base/java.lang.reflect.Field.get(Field.java:444)
[09:36:59] [Server thread/WARN]:        at Sentinel-2.9.1-b525.jar//org.mcmonkey.sentinel.utilities.SentinelNMSHelper.init(SentinelNMSHelper.java:104)
[09:36:59] [Server thread/WARN]:        at Sentinel-2.9.1-b525.jar//org.mcmonkey.sentinel.SentinelPlugin.onEnable(SentinelPlugin.java:472)
[09:36:59] [Server thread/WARN]:        at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:288)
[09:36:59] [Server thread/WARN]:        at io.papermc.paper.plugin.manager.PaperPluginInstanceManager.enablePlugin(PaperPluginInstanceManager.java:202)
[09:36:59] [Server thread/WARN]:        at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.enablePlugin(PaperPluginManagerImpl.java:109)
[09:36:59] [Server thread/WARN]:        at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:520)
[09:36:59] [Server thread/WARN]:        at org.bukkit.craftbukkit.CraftServer.enablePlugin(CraftServer.java:643)
[09:36:59] [Server thread/WARN]:        at org.bukkit.craftbukkit.CraftServer.enablePlugins(CraftServer.java:592)
[09:36:59] [Server thread/WARN]:        at net.minecraft.server.MinecraftServer.loadWorld0(MinecraftServer.java:767)
[09:36:59] [Server thread/WARN]:        at net.minecraft.server.MinecraftServer.loadLevel(MinecraftServer.java:529)
[09:36:59] [Server thread/WARN]:        at net.minecraft.server.dedicated.DedicatedServer.initServer(DedicatedServer.java:326)
[09:36:59] [Server thread/WARN]:        at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1233)
[09:36:59] [Server thread/WARN]:        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:339)
[09:36:59] [Server thread/WARN]:        at java.base/java.lang.Thread.run(Thread.java:1575)
[09:36:59] [Server thread/INFO]: [Sentinel] Sentinel loaded! 
```

**Comment by mcmonkey4eva** [comment]
latest dev build works on 1.21.3

**Comment by SlimeDog** [comment]
Confirmed fixed with Sentinel 2.9.2-b526 with Paper 1.21.3-27. Thanks.
